tests/integration/reports/multi_table/test_quality_report.py

```diff
@@ -299,6 +300,15 @@ def test_quality_report_with_errors():
         'Property': ['Column Shapes', 'Column Pair Trends', 'Cardinality', 'Intertable Trends'],
         'Score': [0.8165079365079364, 0.55, 0.95, 0.5833333333333334],
     })
+
+    pandas_version = version.parse(pd.__version__)
+    if pandas_version >= version.parse('2.2.0'):
```

Comment:
Because different error messages are generated depending on the pandas version. For pandas < 2.2, the KSComplement crashes during rounding, for pandas >= 2.2 `round` doesn't crash anymore, instead the error is raised during `k2_samp`.